### DISCUSSION TITLE
Best Settings for RTX 2070with 8GB VRAM

### DISCUSSION THREAD

**Discussion by ArtaFakhari** [vladmandic/sdnext#762] [Q&A]
Hi,
Do you have any suggestions for the best settings suitable for an RTX 2070 GPU with 8GB of VRAM to get rid of the "CUDA out of memory" error? Currently, I can only use SD with the Tiled VAE extension enabled, but it's only available for text2img and img2img, and not for other sections like training or everywhere else. The system also has 64GB of RAM and an AMD Ryzen 3900X running Ubuntu 22.04.

Thanks in advance.

**Comment by josh5671** [comment]
hello, to be honest.
I don think there any 'trick' to help your situation unless vlad's ui have somehow magically improved the vram call, as i asked the similar question weeks ago.
[https://github.com/vladmandic/automatic/discussions/417#discussioncomment-5704866]


my spec on window;
2700x
32gb ram
rtx3070

**Reply by vladmandic** [reply]
> I can only use SD with the Tiled VAE extension enabled, but it's only available for text2img and img2img, and not for other sections like training or everywhere else

training with large images would create broken training anyhow. model is optimized for 512x512 and while you can play with aspect ratio a bit, using higher resolution for training will not give you any better results. that's why preprocessing training dataset is pretty much a necessity.
